src/utils/message/message-handler.ts

```diff
@@ -45,3 +61,9 @@ export type ResponseMessageFormat<T = string> = BaseMessageFormat & {
 export type ResponseWithMessage<T = string> = {
   [Property in keyof ResponseMessageFormat<T> as Exclude<Property, 'error'>]-?: ResponseMessageFormat<T>[Property]
 }
+
+export function assertMessage<T extends BaseMessageFormat, K extends keyof T>(message: T, method: K): message is T {
```

Comment:
I think `isMessage` is a more common naming convention for custom type guards. Assert may hint raising errors too.

Reply:
It is actually not used anywhere I just left there. I'm going to remove this code.